addons/cartel/CfgFactionClasses.hpp

```diff
@@ -0,0 +1,16 @@
+class CfgFactionClasses {
+    class TACU_Cartel {
+        displayName = "Cartel";
```

Comment:
Are you sure we want it just as "Cartel"? That's very general, is nothing else already using that name?

Reply:
Correct, No other cartel factions currently exist.

Reply:
Quickly reserve it then! :laughing: